### PR TITLE
Added courses Elixir PT-BR

### PR DESCRIPTION
**Pull request by Marcosschwaab**
## What does this PR do?
Add resource(s) ELIXIR PT-BR

### Description
Courses Elixir PT-BR

### Why is this valuable (or not)?
FREE

### How do we know it's really free?
Youtube 
### For book lists, is it a book? For course lists, is it a course? etc.
Video course

### CODE REVIEW

**Review by Navninja7 — COMMENTED** [review]
Hi :) If you add asterisks before all lines and remove blank line at line 100, all your checks will pass surely. Also you could just add the changes instead of deleting and replacing with the new changes, so that changes can be tracked very easily.

File: courses/free-courses-pt_BR.md [hunk 1]
```diff
@@ -94,6 +96,12 @@
 * [Curso de Docker para iniciantes - aprenda Docker em 1 hora](https://www.youtube.com/watch?v=np_vyd7QlXk) - Matheus Battisti
 
 
+### Elixir
+
+* [Curso de Elixir](https://www.youtube.com/watch?v=GXlyLSVDkhI&list=PLydk1OOOmzo8VBeU334j4R4WvSByRNpXR) - ELLY ACADEMY (YouTube)
```
Comment (shrn01) [inline courses/free-courses-pt_BR.md:101]: We support only direct links to the playlist. Please update

File: courses/free-courses-pt_BR.md [hunk 2]
```diff
@@ -94,6 +95,12 @@
 * [Curso de Docker para iniciantes - aprenda Docker em 1 hora](https://www.youtube.com/watch?v=np_vyd7QlXk) - Matheus Battisti
 
 
+### Elixir
+
+* [Curso de Elixir](https://www.youtube.com/playlist?list=PLydk1OOOmzo8VBeU334j4R4WvSByRNpXR) - ELLY ACADEMY (YouTube)
```
Comment (shrn01) [inline courses/free-courses-pt_BR.md:100]:
```suggestion
* [Curso de Elixir](https://www.youtube.com/playlist?list=PLydk1OOOmzo8VBeU334j4R4WvSByRNpXR) - Elly Academy
```

File: courses/free-courses-pt_BR.md [hunk 3]
```diff
@@ -94,6 +95,12 @@
...
+* [Curso de Elixir Alquimia](https://www.youtube.com/playlist?list=PLv3nyCBtlWP8I9rknIrfcJWrO05yEzknD) - Alquimia Stone (YouTube)
```
Comment (shrn01) [inline courses/free-courses-pt_BR.md:101]:
```suggestion
* [Curso de Elixir Alquimia](https://www.youtube.com/playlist?list=PLv3nyCBtlWP8I9rknIrfcJWrO05yEzknD) - Alquimia Stone
```

File: courses/free-courses-pt_BR.md [hunk 4]
```diff
@@ -57,7 +58,7 @@
 
 ### <a id="csharp"></a>C\#
 
-* [C# e Windows Forms: Consultar CEP no WebService dos Correios](https://www.udemy.com/webservice-correios/) - Gilseone Moraes, Training4All Cursos (Udemy)
+* .[C# e Windows Forms: Consultar CEP no WebService dos Correios](https://www.udemy.com/webservice-correios/) - Gilseone Moraes, Training4All Cursos (Udemy)
```
Comment (shrn01) [inline courses/free-courses-pt_BR.md:61]:
```suggestion
* [C# e Windows Forms: Consultar CEP no WebService dos Correios](https://www.udemy.com/webservice-correios/) - Gilseone Moraes, Training4All Cursos (Udemy)
```

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**